docs/monitoring/spm-faq.md

```diff
@@ -1063,16 +1063,16 @@ Note:
 Usage is metered hourly on a per-agent basis. For example:
 
 If you send metrics from a server A to Monitoring App Foo between 01:00 and
-02:00 that's $0.035 for the Standard plan.
+02:00 that's $0.005 for the Standard plan.
```

Comment:
Retention affects the cost, too, not just the plan.  So Standard plan ==> Standard plan & 7 days retention.

Reply:
Added more details about retention.